sql.go

```diff
@@ -71,37 +74,44 @@ func (*SQL) Open(database string, connectionString string) (*dbsql.DB, error) {
 
 // Query executes the provided query string against the database, while
 // providing results as a slice of KeyValue instance(s) if available.
-func (*SQL) Query(db *dbsql.DB, query string, args ...interface{}) ([]keyValue, error) {
-	rows, err := db.Query(query, args...)
-	if err != nil {
-		return nil, err
+func (*SQL) Query(db *dbsql.DB, query string, args ...interface{}) ([]KeyValue, error) {
+	rows, _ := db.Query(query, args...)
+	defer func() {
+		_ = rows.Close()
+	}()
+	if rows.Err() != nil {
+		return nil, rows.Err()
 	}
```

Comment:
Why aren't we checking the error returned by `db.Query()` anymore? I suppose checking and returning `rows.Err()` makes sense, but I don't think those are the same kind of errors. :thinking:

Also, are all these changes triggered by new linter warnings? If not, I would prefer to review them in a separate PR, or at least in a separate commit, so we can easily revert them later if needed.

Reply:
All code changes are due to linter issues now that this PR enables linting. CI changes to enable linting, and the fixes for linting are already a single commit; further isolating a single modification as a separate commit only buys us a little, IMO. Should problems arise, I can easily revert a singular change and reference the original PR.